traffic_ops/app/templates/delivery_service/_form.html.ep

```diff
@@ -399,6 +399,17 @@
 		%= field('ds.global_max_tps')->text(class => 'field', readonly => 'readonly');
 		<% }  %>
 	</div>
+	<div class="block form-row" id="fq_pacing_rate_row">
+		<% unless (field('ds.fq_pacing_rate')->valid) { %>
+			<span class="field-with-error"><%= field('ds.fq_pacing_rate')->error %></span>
+		<% } %>
+		%= label_for 'fq_pacing_rate' => 'Max Bytes per Second allowed per session (3000k or 5M are valid entries)', class => 'label'
```

Comment:
So, if someone writes `3T`, it stores `3000000000` in the DB. And then when someone goes to edit the field a second time, it will just appear as `3000000000`, instead of being displayed as `3T`? That seems odd, but might be consistent with our other behaviour.

Reply:
yeah I noticed that in my testing. it is a little weird but didnt seem overly burdensome on an operator.